redbeams/src/test/java/com/sequenceiq/redbeams/service/stack/DBStackUpdaterTest.java

```diff
@@ -48,8 +48,10 @@ public void testUpdateSslConfigWhenDBStackExistsEnabledAndActiveCertFound() {
         SslConfig sslConfig = new SslConfig();
         sslConfig.setSslCertificateType(SslCertificateType.CLOUD_PROVIDER_OWNED);
         Set<SslCertificateEntry> certificateEntries = new LinkedHashSet<>();
-        certificateEntries.add(new SslCertificateEntry(0, "cloudPlatform", "certPem0", X_509_CERT));
-        certificateEntries.add(new SslCertificateEntry(1, "cloudPlatform", "certPem1", X_509_CERT));
```

Comment:
I guess this change was not intentionally, not related to the PR

Reply:
it is intentional as it was making the tests fail. the constructor wasn't there for the number of parameters being passed in the test cases

Reply:
not sure how it got merged to master

Reply:
also now this change is conflicting with master

Reply:
resolved the conflict